pkg/catalogsourceconfig/cache.go

```diff
@@ -89,7 +89,7 @@ func (c *cache) Evict(csc *v1alpha1.CatalogSourceConfig) {
 
 func (c *cache) Set(csc *v1alpha1.CatalogSourceConfig) {
 	c.entries[csc.ObjectMeta.UID] = &v1alpha1.CatalogSourceConfigSpec{
-		Packages:        csc.Spec.Packages,
+		Packages:        csc.GetPackages(),
```

Comment:
Shouldn't we just leave this as the string we got as input?

Reply:
I suppose technically it shouldn't matter, but it might be better to preserve the behavior for anything other than the simple read.